crates/sui-indexer-alt-jsonrpc/src/data/kv_loader.rs

```diff
@@ -30,8 +34,12 @@ use super::{
 /// - Transactions by digest
 #[derive(Clone)]
 pub(crate) enum KvLoader {
```

Comment:
Better to turn into a struct which has a private `Kind` enum, given most of the fields are shared.

Reply:
(You can disregard this comment because it's superseded by the fact that the metrics and configs probably need to be passed to `BigtableReader`, not `KvLoader`.